CSS/style.css

```diff
@@ -145,9 +186,36 @@ footer {
 .author-text {
   display: block;
   float: left;
+  font-size:
 }
 
-#share-recipe {
+.share-recipe {
   display: inline;
```

Comment:
I'd change this to block or inline-block and it will be easier to style and position.